### PR TITLE
coverage for kyberStorage

### PR DESCRIPTION
**Pull request by ducquangkstn**

### CODE REVIEW

File: test/helper.js [hunk 1]
```diff
@@ -193,6 +193,42 @@ function assertApproximate(val1, val2, errorStr) {
     }
 }
 
+module.exports.assertEqualArray = assertEqualArray;
```
Comment (ducquangkstn) [inline test/helper.js:196]: This is an helper for comparing arrays

File: test/sol5/kyberStorage.js [hunk 2]
```diff
@@ -298,26 +347,63 @@ contract('KyberStorage', function(accounts) {
...
 
     describe("test listing token pair and removing reserve", async() => {
         before("deploy and setup kyberStorage instance & add 1 mock reserve, & 1 mock token", async() => {
             kyberStorage = await KyberStorage.new(admin);
             await kyberStorage.setNetworkContract(network, {from: admin});
             await kyberStorage.setFeeAccountedPerReserveType(true, true, true, false, true, true, {from: admin});
-            //init 1 mock reserve
-            let result = await nwHelper.setupReserves(network, [], 1,0,0,0, accounts, admin, operator);
+            //init 2 mock reserve
+            let result = await nwHelper.setupReserves(network, [], 2,0,0,0, accounts, admin, operator);
```
Comment (ducquangkstn) [inline test/sol5/kyberStorage.js:400]: This is for coverage at line https://github.com/KyberNetwork/smart-contracts/blob/bdcf45724c32115ce2d7ce66bb0461be1fd82ab6/contractsSol5/KyberStorage.sol#L363

File: test/sol5/kyberStorage.js [hunk 3]
```diff
@@ -215,19 +258,25 @@ contract('KyberStorage', function(accounts) {
...
                 "max proxies limit reached"
             );
+
+            proxies = await kyberStorage.getKyberProxies();
+            Helper.assertEqualArray(proxies, [proxy1, proxy2], "unexpected proxies");
         });
 
-        if("test removeKyberProxy revert if not added", async() => {
+        it("test remove proxy revert if not added", async() => {
+            await kyberStorage.addKyberProxy(proxy2, maxProxies, {from: network});
```
Comment (ducquangkstn) [inline test/sol5/kyberStorage.js:273]: this is for coverage at https://github.com/KyberNetwork/smart-contracts/blob/bdcf45724c32115ce2d7ce66bb0461be1fd82ab6/contractsSol5/KyberStorage.sol#L183

File: test/helper.js [hunk 4]
```diff
@@ -193,6 +193,42 @@ function assertApproximate(val1, val2, errorStr) {
...
+        }           
+        else if (this[i] != array[i]) { 
+            // Warning - two different object instances will never be equal: {x:20} != {x:20}
+            return false;   
+        }           
+    }       
+    return true;
+}
+// Hide method from for-in loops
+Object.defineProperty(Array.prototype, "equals", {enumerable: false});
```
Comment (ilanDoron) [inline test/helper.js:230]: why do we hide the method here?

File: test/sol5/kyberStorage.js [hunk 5]
```diff
@@ -149,6 +153,29 @@ contract('KyberStorage', function(accounts) {
...
+                kyberStorage.setContracts(feeHandlerAddr, matchingEngineAddr, { from: operator}), "only network"
+            );
+
+            await expectRevert(
+                kyberStorage.setContracts(feeHandlerAddr, matchingEngineAddr, { from: admin}), "only network"
+            );
+        });
+
+        it("should have network set contracts", async() => {
+            await kyberStorage.setDAOContract(DAOAddr, { from: network});
```
Comment (ilanDoron) [inline test/sol5/kyberStorage.js:176]: please validate here it was set

File: test/helper.js (hunk 4 above)
Comment (ducquangkstn) [reply test/helper.js:230]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Enumerable_attribute
When use for-in like below, the `equals` will not appear
```
for (var i in [1,2,3]) {
  console.log(i);
}
```

**Pull request merged**